src/Database/Schema/SqliteSchemaDialect.php

```diff
@@ -294,7 +294,6 @@ public function convertIndexDescription(TableSchema $schema, array $row): void
         foreach ($statement->fetchAll('assoc') as $column) {
             $columns[] = $column['name'];
         }
-        $statement->closeCursor();
```

Comment:
We could clean up code like above with `array_column()`.

Reply:
Can be done in a follow up PR.